docs/insights-archive-sample/config/installers.datahub.sap.com/datahubs/sdi/default.json

```diff
@@ -0,0 +1 @@
  \"description\": \"Modeler: Base registry for pulling images\",\n  \"type\": \"string\",\n  \"defaultValue\": \"container-image-registry-sdi-observer.apps.lab.hpecloud.org\"\n}\n","vrepPath":"vsystem/parameters/vflow.baseRegistry.json"}],"version":"3.1.13-ovmhx"}},{"name":"data-tools-ui"},{"name":"license-manager"},{"name":"vora-tools"},{"name":"vrelease-appmanagement"},{"name":"vrelease-delivery-di"},{"name":"vrelease-diagnostics"},{"name":"vsolution-app-base"},{"name":"vsolution-app-base-db"},{"name":"vsolution-app-data"},{"name":"vsolution-dh-flowagent"},{"name":"vsolution-shared-ui"},{"name":"vsolution-vsystem-ui"},{"name":"dsp-release-onprem"},{"name":"dsp-release-onprem-system"}],"tenants":[{"name":"system","strategyName":"sdi-system-extension-strategy","tenantAdminCreds":"vora.conf.initial.user","vsystemObjects":["{\n  \"kind\": \"Route\",\n  \"apiVersion\": \"v1\",\n  \"metadata\": {\n    \"name\": \"diagnostics-grafana\"\n  },\n  \"status\": {\n    \"ingress\": [\n      {\n        \"host\": \"diagnostics-grafana\",\n        \"port\": 443,\n        \"tls\": true\n      }\n    ]\n  }\n}\n","{\n  \"kind\": \"Route\",\n  \"apiVersion\": \"v1\",\n  \"metadata\": {\n    \"name\": \"diagnostics-kibana\"\n  },\n  \"status\": {\n    \"ingress\": [\n      {\n        \"host\": \"diagnostics-kibana\",\n        \"port\": 443,\n        \"tls\": true\n      }\n    ]\n  }\n}\n"]},{"name":"default","strategyName":"sdi-default-extension-strategy","tenantAdminCreds":"vora.conf.tenants.defaultadmin","vsystemSecrets":["vflow-secret"]}],"vRep":{},"vSystem":{"auditlogAddress":"https://auditlog:3030","clusterAdminHostnames":"vora-tx-coordinator,diagnostics-prometheus-server,storagegateway","features":{"apiGateway":{}},"serviceType":"ClusterIP"},"vflow":{"enabled":true}}},"status":{"conditions":[{"message":"Operation in progress","operationId":"3a248d","package":"3.1.13","reason":"preInstall","status":"False","timestamp":"2021-01-25T16:47:32Z","type":"inProgress"},{"message":"preInstall hooks successfully executed","operationId":"3a248d","package":"3.1.13","status":"True","timestamp":"2021-01-25T16:47:36Z","type":"preInstall"},{"message":"postInstall hooks successfully executed","operationId":"3a248d","package":"3.1.13","status":"True","timestamp":"2021-01-25T16:47:39Z","type":"postInstall"},{"message":"Operation in progress","operationId":"c03db8","package":"3.1.13","previousPackage":"3.1.13","reason":"preRunLevelStarted","status":"False","timestamp":"2021-01-25T16:47:43Z","type":"inProgress"},{"message":"preRunLevelStarted hooks successfully executed","operationId":"c03db8","package":"3.1.13","status":"True","timestamp":"2021-01-25T16:47:44Z","type":"preRunLevelStarted"},{"message":"postRunLevelStarted hooks successfully executed","operationId":"c03db8","package":"3.1.13","status":"True","timestamp":"2021-01-25T16:47:48Z","type":"postRunLevelStarted"}],"expiresAt":"2021-01-25T17:47:51Z","reason":"Expired","startedAt":"2021-01-25T16:47:51Z","state":"Failed"}}
```

Comment:
Pretty format please.

Reply:
Oh, I didn't notice we did that in the sample archive. I thought we kept things authentic.